opal/core/api.py

```diff
@@ -390,32 +390,36 @@ def retrieve(self, request, pk=None):
         return json_response(patientlist.to_dict(request.user))
 
 
-router.register('patient', PatientViewSet)
-router.register('episode', EpisodeViewSet)
-router.register('record', RecordViewSet)
-router.register('userprofile', UserProfileViewSet)
-router.register('user', UserViewSet)
-router.register('tagging', TaggingViewSet)
-router.register('patientlist', PatientListViewSet)
-router.register('patientrecordaccess', PatientRecordAccessViewSet)
+def register_plugin_apis():
+    for plugin in plugins.OpalPlugin.list():
+        for api in plugin.get_apis():
+            router.register(*api)
 
-router.register('referencedata', ReferenceDataViewSet)
-router.register('metadata', MetadataViewSet)
 
-for subrecord in subrecords():
-    sub_name = subrecord.get_api_name()
+def register_subrecords():
+    for subrecord in subrecords():
+        sub_name = subrecord.get_api_name()
 
-    class SubViewSet(SubrecordViewSet):
-        base_name = sub_name
-        model     = subrecord
+        class SubViewSet(SubrecordViewSet):
+            base_name = sub_name
+            model     = subrecord
 
-    router.register(sub_name, SubViewSet)
+        router.register(sub_name, SubViewSet)
 
 
-def register_plugin_apis():
-    for plugin in plugins.OpalPlugin.list():
-        for api in plugin.get_apis():
-            router.register(*api)
+def initialise_router():
```

Comment:
14 instances of initialise and 18 instances of initialize in our codebase. 
We should pick one.